mzroll/tabledockwidget.cpp

```diff
@@ -2389,3 +2282,155 @@ MatrixXf TableDockWidget::getGroupMatrix(vector<mzSample*>& samples, PeakGroup::
     return X;
 }
 
+QWidget* TableToolBarWidgetAction::createWidget(QWidget *parent) {
+
+
+    if (btnName == "titlePeakTable") {
+ 
+        QLabel *titlePeakTable = new QLabel(parent);
```

Comment:
Declaring titlePeakTable here can create conflicts because earlier it was declared in the TableDockWidget's constructor and used for title of peak table present in scatterplot.